From mithlesh@linsyssoft.com  Fri Apr  3 14:51:31 2009
From: Mithlesh Thukral <mithlesh@linsyssoft.com>
Date: Wed, 25 Mar 2009 15:52:54 +0530 (IST)
Subject: Staging: sxg: slicoss: Specify the license for Sahara SXG and Slicoss drivers
To: Greg Kroah-Hartman <gregkh@suse.de>
Cc: Sahara Project <saharaproj@linsyssoft.com>, Christopher Harrer <charrer@alacritech.com>
Message-ID: <alpine.LFD.2.00.0903251551590.28224@localhost.localdomain>


Specify a license for the Alacritech Sahara (10Gbe) and SLICOSS (1GBE)
firmware files.

Signed-off-by: Christopher Harrer <charrer@alacritech.com>
Signed-off-by: Mithlesh Thukral <mithlesh@linsyssoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 firmware/WHENCE |   33 ++++++++++++++++++++++++++++++---
 1 file changed, 30 insertions(+), 3 deletions(-)

--- a/firmware/WHENCE
+++ b/firmware/WHENCE
@@ -372,7 +372,21 @@ File: slicoss/oasisdbgdownload.sys.ihex
 File: slicoss/oasisdownload.sys.ihex
 File: slicoss/oasisrcvucode.sys.ihex
 
-Licence: Unknown
+Licence:
+		Copyright (C) 1999-2009 Alacritech, Inc.
+
+	as an unpublished work. This notice does not imply unrestricted or
+	public access to the source code from which this firmware image is
+	derived.  Except as noted below this firmware image may not be
+	reproduced, used, sold or transferred to any third party without
+	Alacritech's prior written consent.  All Rights Reserved.
+
+	Permission is hereby granted for the distribution of this firmware
+	image as part of a Linux or other Open Source operating system kernel
+	in text or binary form as required.
+
+	This firmware may not be modified and may only be used with
+	Alacritech hardware.
 
 Found in hex form in kernel source.
 
@@ -383,10 +397,23 @@ Driver: SXG - Alacritech IS-NIC products
 File: sxg/saharadownloadB.sys.ihex
 File: sxg/saharadbgdownloadB.sys.ihex
 
-Licence: Unknown
+Licence:
+		Copyright (C) 1999-2009 Alacritech, Inc.
+
+	as an unpublished work. This notice does not imply unrestricted or
+	public access to the source code from which this firmware image is
+	derived.  Except as noted below this firmware image may not be
+	reproduced, used, sold or transferred to any third party without
+	Alacritech's prior written consent.  All Rights Reserved.
+
+	Permission is hereby granted for the distribution of this firmware
+	image as part of a Linux or other Open Source operating system kernel
+	in text or binary form as required.
+
+	This firmware may not be modified and may only be used with
+	Alacritech hardware.
 
 Found in hex form in kernel source.
-Copyright 1997-2009 Alacritech, Inc. All rights reserved.
 
 --------------------------------------------------------------------------
 
